prog/weaver/main.go

```diff
@@ -69,6 +71,7 @@ func main() {
 	flag.IntVar(&bufSzMB, "bufsz", 8, "capture buffer size in MB")
 	flag.StringVar(&httpAddr, "httpaddr", fmt.Sprintf(":%d", weave.HTTPPort), "address to bind HTTP interface to (disabled if blank, absolute path indicates unix domain socket)")
 	flag.StringVar(&iprangeCIDR, "iprange", "", "IP address range to allocate within, in CIDR notation")
+	flag.StringVar(&defltSubnet, "defaultsubnet", "", "default subnet for IP address allocation, in CIDR notation")
```

Comment:
How about `-iprange` and `-ipsubnet`, with the latter defaulting to the former?  This would mean that anyone using the feature as shipped in v0.11 could continue unchanged.

Also, as noted elsewhere, anyone wanting to give some containers manually-assigned addresses in the same subnet as ipam-assigned addresses would use a default subnet _larger_ than iprange, so "subnet of xxx" is not strictly the right phrasing.